docs/integrations/open_telemetry/index.md

```diff
@@ -0,0 +1,195 @@
+# OpenTelemetry
+
+Mirascope provides out-of-the-box integration with [OpenTelemetry](https://opentelemetry.io/docs/what-is-opentelemetry/).
+
+## Setting up Manual Instrumentation
+
+Due to the detailed about of data Mirascope needs to collect about calls, we use manual instrumentation. Thankfully, it is low-code on the user-end so Mirascope still offers convenience.
+
+First thing to do is install mirascope[opentelemetry]
+
+```bash
+pip install mirascope[opentelemetry]
+```
+
+### The Setup
```

Comment:
nit: I think it might be cleaner if this and the with mirascope section are just ## H2 sections? instead of being nested. We can probably just drop the "Setting up Manual Instrumentation" header (or make "The Setup" that header instead).